app/questionnaire/rules.py

```diff
@@ -69,6 +69,20 @@ def evaluate_repeat(repeat_rule, answer_store):
         return repeat_function(filtered)
 
 
+def evaluate_skip_condition(skip_condition, answer_store):
```

Comment:
I think this method name should be more generic. Perhaps something like evaluate_when_list, given it doesn't just apply to skip_conditions, it's equally applicable to the when attribute of any rule.